### ISSUE TITLE
Request: Unique ID

### ISSUE THREAD

**Issue by EskoBrummel** [suryajasper/Food-Bank-Smart-Routes#8]
Output addresses seem to be slightly altered from input, this complicates using tools like Vlookup to associate other patient info with the out altered addresses. Would be great if addresses could also be entered with an ID that keeps any altered output associated with the original input address as well as enables ease associating other relational data sets to stay connected via the ID.

**Comment by EskoBrummel** [comment]
Question for Joyce: Would this issue be solved if we simply didn't alter the output address (i.e. remember that association between the input and output)? 

SB: Yeah that could work. I assumed that google was adjusting the addresses to work with their mapping. If the addresses don't need to be changed then this problem would be fixed. 

Some context: addresses come from a list created by a number of people and we don't yet have a great way of making sure they enter the addresses in a uniform way. If it would help for us to make sure the addresses are entered in a clear, uniform way, let us know and we can figure out a way to make that happen.

**Comment by kyleCampbel1** [comment]
If there is only one dataset, would it be an issue to provide the entire patient spreadsheet? If that came in a uniform format, we could likely output routes with the associated information you want. If there are multiple data sets, then ID or not changing address would be best, I agree.

**Comment by ninchkn78** [comment]
Yeah I think this depends on how we want this service to scale especially to other orgs that may be using it. So something to consider is if we think it would make sense for anyone using this to always be also entering in info associated with the addresses.
